### ISSUE TITLE
Select field ordering

### ISSUE THREAD

**Issue by afeld** [afeld/jeditable-rails#1]
There should be a way to specify the ordering of select options passed in as a hash, either through `:data` or another param.  Perhaps accepting two-dimensional array to `:data` instead?

```
editable_field @user, :zip_code, :data => [['visa', 'Visa'], ['mastercard', 'Mastercard'], ['amex', 'American Express']]
```

**Comment by tranquiliste** [comment]
Agree, Option are sorted by the key, not in the order you provide the data. 
Any idea on what to modify to make the order in the data the order in the select?
